tests/test_skos.py

```diff
@@ -537,46 +522,46 @@ def _get_sortlabel(self):
         return Label('allereerste', type='sortLabel', language='nl-BE')
 
     def test_label_empty(self):
-        self.assertEqual(None, label([]))
-        self.assertEqual(None, label([], 'nl-BE'))
-        self.assertEqual(None, label([], None))
-        self.assertEqual(None, label([], 'und'))
-        self.assertEqual(None, label([], ['nl-BE']))
+        assert None == label([])
```

Comment:
All None equalities, or inequalities should probably become `is None` or `is not None`